src/js/components/Video/index.d.ts

```diff
@@ -11,7 +11,29 @@ export interface VideoProps {
   a11yTitle?: A11yTitleType;
   alignSelf?: AlignSelfType;
   autoPlay?: boolean;
-  controls?: false | 'over' | 'below';
+  controls?:
+    | false
+    | 'over'
+    | 'below'
+    | {
+        position?: false | 'over' | 'below';
+        items?: [
+          (
+            | 'closedCaption'
+            | 'configure'
+            | 'fullScreen'
+            | 'pause'
+            | 'play'
+            | 'reduceVolume'
+            | 'volume'
```

Comment:
I suspect these names were selected based on the icon name. But, I don't think these are quite right as control names. Instead, I think we should use: captions, fullScreen, play, pause, volume. I don't think we need "configure" as a built in control. I think we should combine the "volume" into a single name here but then expand that to multiple controls automatically internally.